### PR TITLE
Method to get one news from db

### PR DESCRIPTION
**Pull request by Kukharchuk**

### CODE REVIEW

File: database/database.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+  password,
+  port,
+  dbName,
+  host,
 } = require('../configs/config.json');
 
 class DB {
   constructor() {
-    this.URI = `mongodb://${user}:${password}@${host}:${port}/${dbName}`;
+    this.URI = 'mongodb://${user}:${password}@${host}:${port}/${dbName}';
```
Comment (demns) [inline database/database.js]: no way it's going to work

File: server.js [hunk 2]
```diff
@@ -4,17 +4,19 @@ const app = express();
...
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
 app.use(bodyParser.json());
 
 const port = process.env.PORT || 8080;
 
 app.use((req, res, next) => {
-  res.header('Access-Control-Allow-Origin', 'http://localhost:3000');
+  res.header('Access-Control-Allow-Origin', '*');
```
Comment (demns) [inline server.js]: all right. any problems with this approach?)

Comment (Kukharchuk) [reply server.js]: Maybe it's not very good idea to use "*", but we still need it when deploying to heroku, can i leave so?

File: database/database.js (hunk 1 above)
Comment (Kukharchuk) [reply database/database.js]: Yes, but I had some mistekes when trying to connect to db and change quotes and then even fogot about them

**Pull request merged**